openapi-core/src/main/java/io/ballerina/openapi/core/generators/client/model/OASClientConfig.java

```diff
@@ -77,8 +79,7 @@ public static class Builder {
         private boolean resourceMode = true;
         private boolean isPlugin = false;
 
-        private String license = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n\n" +
-                "// This file is auto-generated by the Ballerina OpenAPI tool.\n";
+        private String license = DO_NOT_MODIFY_FILE_HEADER;
```

Comment:
Do need to have this `license` variables? can't we directly use the `DO_NOT_MODIFY_FILE_HEADER` instead?

Reply:
This variable was created with the purpose of any modification to done license with file headers. will check and remove